tag_release.py

```diff
@@ -0,0 +1,554 @@
+#!/usr/bin/env python
+"""
+Tag repos for an Open edX release.
+"""
+from __future__ import unicode_literals, print_function
+import sys
+import re
+import json
+import getpass
+import argparse
+import logging
+try:
+    from path import Path as path
+    from git import Repo, Commit
+    from git.refs.symbolic import SymbolicReference
+    from git.exc import GitCommandError
+    import requests
+    from requests.exceptions import RequestException
+    import yaml
+except ImportError:
+    print("Error: missing dependencies! Please run this command to install them:")
+    print("pip install path.py requests GitPython PyYAML")
+    sys.exit(1)
+
+log = logging.getLogger(__name__)
+
+# fetch repo list
+# for each repo, find the commit that should be tagged
+#   "release" branch for edx-platform
+#   "master" for most others
+#   whatever edx-platform points to for xblock and ora2
+# show commits and human-readable refs (branch name, tag name, etc)
+#   ask for confirmation
+# create tags on GitHub
```

Comment:
This description (perhaps updated to mention repos.yaml) would be better in the docstring.